okhttp/src/main/java/io/grpc/transport/okhttp/OkHttpClientTransport.java

```
@@ -476,6 +485,9 @@ public void run() {
         }
       } catch (IOException ioe) {
         abort(ioe);
+      } catch (RuntimeException e) {
```

Just catch all Exceptions or Throwables?